index.js

```diff
@@ -132,6 +132,9 @@ module.exports = class KmlStream extends Transform {
       let thing
       if (!data.trim()) return
       switch (this.currentTag.name) {
+        case 'styleurl':
+          // ignore
```

Comment:
@calvinmetcalf No because the behavior after this switch statement is to add any unhandled tags within a placemark as a property, the easiest way to ignore tags is to handle it and do nothing - open to suggestions to reformat this logic though.

Reply:
ok I see now, maybe it would make sense to put the stuff after the switch statement into the default case to make it a little more obvious